src/components/VariantSearchField.tsx

```diff
@@ -101,36 +248,37 @@ export const VariantSearchField = ({ onVariantSelect, currentSelection, triggerS
   const suggestMutations = (query: string): string[] => {
     // TODO Fetch all/common known mutations from the server
     // For now, just providing a few mutations so that the auto-complete list is not entirely empty.
-    return [
+    let notation = query.toLowerCase().startsWith('n') ? 'nsp' : query.startsWith('orf1ab') ? 'orf1ab' : '';
+    let options: string[] = [
       'S:D614G',
-      'ORF1b:P314L',
+      `ORF1b:P314L`,
```

Comment:
To keep the PR clean, please try to avoid unnecessary edits and revert the changes from ' to `